ui/src/app/layout/dashboard/dashboard.component.ts

```diff
@@ -102,10 +149,20 @@ export class DashboardComponent implements OnInit {
         });
     }
 
+    checkProfilePictureExists(image: HTMLImageElement, change: ChangelogEntry): void {
+        const h = image.naturalHeight;
+        const w = image.naturalWidth;
+        if (h === 1 && w === 1 && image.src !== change.defaultPictureUrl) {
+            image.src = change.defaultPictureUrl;
```

Comment:
Instead of accessing img directly, you can set `change.profilePictureUrl = change.defaultPictureUrl` here.